dotnetcoreVolunteer/Models/Achievments.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
```

Comment:
I would change the name of the file to be "Achievements.cs" incase someone is expecting that name.

Reply:
Change this file to Achievement.cs
Entities should be singular